app/test_split_integer.py

```diff
@@ -2,20 +2,37 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 6
+    number_of_parts = 2
```

Comment:
use numbers directly in assert, no need for extra variables

fix in all places